lex/lexer.mll

```diff
@@ -137,7 +137,7 @@ rule main = parse
       main lexbuf }
   | "#" [' ' '\t']* (['0'-'9']+ as num) [' ' '\t']*
     ('\"' ([^ '\010' '\013' '\"']* as name) '\"')?
-    [^ '\010' '\013']* '\010'
+    [^ '\010' '\013']* '\013'? '\010'
```

Comment:
If we want to ignore _all_ CRs, then it would be better as follows:
```suggestion
    [^ '\010' '\013']* '\013'* '\010'
```

Reply:
You're right. will be fixed too.